app/web/src/features/messages/utils.ts

```diff
@@ -59,7 +59,8 @@ export function groupChatTitleText(
         .join(", ");
 }
 
-export function groupChatUsernameText(
+/** Returns the other user's username, or null if there are more than 2 users. */
+export function formatGroupChatUsernames(
```

Comment:
```suggestion
export function getDmUsername(
```
I'd suggest renaming the function to this then and remove the need of the comment